docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Runtime package store
+description: Runtime package store and target manifests
+keywords: .NET, .NET Core
+author: beleroy
+ms.date: 04/13/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+---
+
+# Runtime package store
+
+Starting with .NET Core 2.0, it is possible to package and deploy applications against a known set of packages that is known to exist on the target environment. The benefits in doing so is that it is possible to deploy smaller published applications, to use less disk space, and in some cases to improve startup performance.
```

Comment:
it is -> it's (always use contractions)
a known set of packages that is known: very repetitive use of known. I recommend you trimming the second one

re. benefits
It is possible or do you definitely get those benefits? 
deploy smaller published apps? Perhaps it should say just deploy smaller apps.